.cirrus.yml

```diff
@@ -166,3 +166,41 @@ static_binary_task:
     save_cache_script: |
         mv /nix .cache
         chown -Rf $(whoami) .cache
+
+# Build the static binary for arm64
+static_binary_arm64_task:
+    depends_on:
+        - 'config'
+        - 'fmt'
+
+    gce_instance:
+        image_name: "${FEDORA_CACHE_IMAGE_NAME}"
+        cpu: 8
+        memory: 12
```

Comment:
Ouch, this could get expensive for us if it runs multiple times per PR and fails after 2-hours.  Would it make sense/be safer to limit this task to only branches?  Another idea could be to add a `timeout_in: 30m` attribute (or similar).  What do you think?

Reply:
Added a 20 minute timeout. With using cachix this should be totally enough.